testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -4,6 +4,8 @@
 import torch.nn.init as init
 import torch.nn as nn
 import torch.nn.functional as F
+import tensorflow as tf
+import tf2onnx
```

Comment:
Please, leave a comment with versions of TensorFlow and TF2ONNX.

Reply:
Added.  Sorry, I misread your initial comment.